nixos/tests/nextcloud/basic.nix

```diff
@@ -9,7 +9,23 @@ in {
 
   nodes = {
     # The only thing the client needs to do is download a file.
-    client = { ... }: {};
+    client = { pkgs, ... }: {
+      environment.systemPackages = [ pkgs.hello ];
```

Comment:
I assume this isn't needed, right?

Reply:
Lol, indeed it isn't :laughing: Updated.